src/pages/index.js

```diff
@@ -65,6 +66,7 @@ const IndexPage = ({ data, pageContext }) => {
             className={cx(styles.intro, 'intro-text')}
           >
             <div className={styles.introText}>
+              <CountdownContainer />
```

Comment:
The way the countdown timer is positioned on the page, I expect it to be a countdown to the event itself, not the submission deadline. I suggest moving this under the paragraph which talks about submitting a proposal.